ibis_vega_transform/vegaexpr.py

```diff
@@ -11,11 +11,61 @@
 
 import ibis
 import ibis.expr.types as it
-import pytz
 from mypy_extensions import TypedDict
 from typing_extensions import Literal
 
 from altair_transform.utils import evaljs
+import altair_transform.utils._evaljs
+
+# Monkey patch altair_transform so that boolean operators  work on ibis expression
```

Comment:
That's kind of unfortunate that we have to do this. It looks like there are about 20 operators in the module -- do you think we'll have to cover all of them?

Reply:
Or might it be worthwhile to try to go a bit deeper into the `altair_transform` module and pull out the parser without the operators?

Reply:
Good point. I will take a look at the other operators to see if they work as is, or need to be patched as well. 

> Or might it be worthwhile to try to go a bit deeper into the altair_transform module and pull out the parser without the operators?

Yeah, that sounds good, long term. You OK with punting that for future work? Or would you like me to look into it for this PR?

Reply:
It looks like the other comparison operators (`<`, ...) work as is,  I am not sure if/how the [bitwise operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators) are used in vega  expressions, so I am not sure how they should be translated.  I am inclined to add more overrides if we come across use cases which require them in the future.